tests/catalyst/contrib/datasets/test_movielens_20m.py

```diff
@@ -0,0 +1,170 @@
+import os
+import shutil
+
+import pytest
+
+from catalyst.settings import SETTINGS
+
+if SETTINGS.ml_required and SETTINGS.is_torch_1_7_0:
+    from catalyst.contrib.datasets import MovieLens20M
+
+minversion = pytest.mark.skipif(
+    not (SETTINGS.is_torch_1_7_0), reason="No catalyst[ml] required or torch version "
+)
+
+
+def setup_module():
+    """
+    Remove the temp folder if exists
+    """
+    data_path = "./data"
+    try:
+        shutil.rmtree(data_path)
+    except Exception as e:
+        print("Error! Code: {c}, Message, {m}".format(c=type(e).__name__, m=str(e)))
+
+
+@minversion
+@pytest.mark.skipif(not (SETTINGS.ml_required), reason="No catalyst[ml] required")
+def test_download_split_by_user():
+    """
+    Test movielense download
+    """
+    MovieLens20M("./tmp_data", download=True, sample=True)
```

Comment:
could we use TemporaryDirectory? like https://github.com/catalyst-team/catalyst/blob/e1d78b7dd568c4d5cd94be417d6d8b93ef27b2e5/tests/catalyst/runners/test_train_flags.py